frontend/mono_frontend/src/pages/AccountList/CardAccounts.jsx

```diff
@@ -0,0 +1,81 @@
+import { useRouteLoaderData } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { BACKEND_URL } from '../../config/envs'
+import './CardAccounts.css'
+
+export const getCards = async function (token) {
+	const endpoint = `${BACKEND_URL}/monobank/monocards`
+
+	const response = await fetch(endpoint, {
+		method: 'GET',
+		headers: {
+			'Content-Type': 'application/json',
+			Authorization: `Bearer ${token}`
+		}
+	})
+	const data = await response.json()
+
+	if (!response.ok) {
+		return null
+	}
+	return data
+}
+
+export const CardAccounts = () => {
+	const token = useRouteLoaderData('token')
+	const [cardData, setCardData] = useState(null)
+
+	useEffect(() => {
+		const fetchData = async function () {
+			try {
+				const access = await getCards(token)
+				setCardData(access)
+			} catch (error) {
+				console.error('Error fetching card data:', error)
+			}
+		}
+
+		let ignore = fetchData()
+	}, [token])
+
+	return (
+		<>
+			{cardData && (
+				<div>
+					<table className='table'>
+						<thead className='table__head'>
+							<tr key={1} className='table__row'>
+								<th className='table__title'>№</th>
+								<th className='table__title'>AccountNumber</th>
+								<th className='table__title'>Balance</th>
+								<th className='table__title'>Currency</th>
+								<th className='table__title'>Count Card</th>
+								<th className='table__title'>Number Cards</th>
+							</tr>
+						</thead>
+						<tbody className='table__body'>
+							{cardData.map(
+								({ id, iban, type, currency, balance, masked_pan }, index) => (
+									<tr className='table__body_tr' key={index}>
+										<td className='table__description'>{index + 1}</td>
+										<td className='table__description'>{iban}</td>
+										<td className='table__description'>{balance / 100}</td>
```

Comment:
Also consider credit limit. It is also important information to show in cards overiview. Lets try to highlight it. (with extra brackets, color, extra column, ect...)

Reply:
Done